src/overview/bare-metal.md

```diff
@@ -1,59 +1,59 @@
+[embedded-rust-book-no-std]: https://docs.rust-embedded.org/book/intro/no-std.html
+[rust-lib-core]: https://doc.rust-lang.org/core/index.html
+[rust-lib-std]: https://doc.rust-lang.org/std/index.html
 
-Previously, the primary focus of `no_std` development was the ESP32 and (to a lesser extent) the ESP8266.
 
-Now there is a renewed effort to implement `no_std` support for the entire lineup of Espressif devices from the ESP32 and newer. These new HALs can be found in the [esp-rs/esp-hal] repository.
+## Current support
 
-There is also some level of support for Wi-Fi and Bluetooth via [esp-rs/esp-wifi] for ESP32, ESP32-C3, ESP32-S2, and ESP32-S3.
+The table below covers the current support for `no_std` at this moment for different Espressif products.
 
-[esp-rs/esp-hal]: https://github.com/esp-rs/esp-hal
-[esp-rs/esp-wifi]: https://github.com/esp-rs/esp-wifi
+> **Notes**:
+>
+> - ✅ - the feature is implemented or supported
+> - ⏳ - the feature is under development
+> - ❌ - the feature is not supported
+> - [ESP8266 HAL][esp-rs/esp8266-hal] is in maintenance mode and no further development will be done for this chip
 
-## Chip Support
+|         | [HAL][esp-rs/esp-hal] | [PAC][esp-rs/esp-pacs] | [Wi-Fi][esp-rs/esp-wifi] | [Bluetooth][esp-rs/esp-wifi] | [Heap][esp-rs/esp-alloc] | [Println][esp-rs/esp-println] | [Backtrace][esp-rs/esp-backtrace] | [Storage][esp-rs/esp-storage] |
+| -------- | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+|  ESP32   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-C2 |   ✅   |   ✅   |   ✅   |   ✅   |   ❌   |   ✅   |   ✅   |   ✅   |
+| ESP32-C3 |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-C6 |   ✅   |   ✅   |   ✅   |   ⏳   |   ❌   |   ✅   |   ✅   |   ❌   |
+| ESP32-S2 |   ✅   |   ✅   |   ✅   |   ❌   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-S3 |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |
+| ESP32-H2 |   ❌   |   ✅   |   ❌   |   ❌   |   ❌   |   ✅   |   ❌   |   ❌   |
+| ESP8266  |   ✅   |   ✅   |   ❌   |   ❌   |   ❌   |   ✅   |   ❌   |   ❌   |
 
-Chip support for `no_std` requires LLVM/Clang support just like for `std`. However, this has no dependency on `esp-idf`. In addition to compiler support, it's necessary to have peripheral access crates (PAC) and hardware abstraction layers (HAL) for your desired chip.
+Here is a list with the relevant crates and small summary of what they do for `no_std`:
 
-Refer to the table below to see if your chip is supported. Please note that the `no_std` HALs are still in the early phases of development, so not all peripherals have had drivers implemented.
+| Symbol | Repository | Description |
+| ------ | ---------- | ----------- |
+| **HAL** | [esp-rs/esp-hal] | Hardware abstraction layer |
+| **PAC** | [esp-rs/esp-pacs] | Peripheral access crates |
+| **Wi-Fi** | [esp-rs/esp-wifi] | Wi-Fi support |
+| **Bluetooth** | [esp-rs/esp-wifi] | Bluetooth LE support |
```

Comment:
Not sure what the "Symbol" column is for? Wi-Fi/Bluetooth use the same repo though, weird to link it twice, and ESP-NOW isn't mentioned at all (which is also supported by that repo).

Reply:
We had a conversation around this in this comment: https://github.com/esp-rs/book/pull/75#discussion_r1153286243

Reply:
Delete the Symbol column